README.md

```diff
@@ -15,10 +15,23 @@ Tanzu Toolkit for Visual Studio is an extension for Visual Studio (2022 & 2019)
 - [Remotely debug](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/wiki/Remote-Debugging-TAS-Apps) apps running on TAS within Visual Studio
 
 ## Install
-- To install this extension, you will need Visual Studio 2022 or 2019.
-  - The extension can then be found on both versions' Visual Studio Marketplace, by searching for Tanzu Net Experience.
-- Download the `Tanzu.Toolkit.VisualStudio.vsix` file & double-click to prompt an installation into your Visual Studio instance.
-- To uninstall this extension from Visual Studio, visit the Visual Studio Extension Manager found under the `Extensions` menu.
+To install this extension, you will need:
+  1. Visual Studio version 2022 or 2019
+  2. The `Tanzu.Toolkit.VisualStudio.vsix` extension file, downloadable from any of these sources:
+      - The Visual Studio Marketplace:
+        - through the IDE by searching "Tanzu" in the "Manage Extensions" window
+        - through the website: [VS 2022](https://marketplace.visualstudio.com/items?itemName=TanzuNETExperience.TanzuToolkitForVisualStudio2022), [VS 2019](https://marketplace.visualstudio.com/items?itemName=TanzuNETExperience.TanzuToolkitForVisualStudio2019)
+      - vsixgallery.com: [VS 2022](https://www.vsixgallery.com/extension/TanzuToolkitForVisualStudio2022.ff7b6f3e-0410-4ff9-a40a-a719ee9da901), [VS 2019](https://www.vsixgallery.com/extension/TanzuToolkitForVisualStudio2019.ff7b6f3e-0410-4ff9-a40a-a719ee9da901)
+      - The ["Releases" section](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/releases) of this repository
+        - look for the `.vsix` file under "Assets" that corresponds to your version of Visual Studio
+
+Once you've downloaded `Tanzu.Toolkit.VisualStudio.vsix`, double-click it to prompt an installation into your instance of Visual Studio.
```

Comment:
This won't apply if you install through the IDE

Reply:
```suggestion
If the process you selected involves saving `Tanzu.Toolkit.VisualStudio.vsix` to disk, double-click it to prompt an installation into your instance of Visual Studio.
```

Reply:
Oops didn't see this suggestion

Reply:
I rephrased the whole section in this commit: [6f26e8e](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/pull/339/commits/6f26e8ec1e9c6626811da2f18c791db4deb3c9a7) -- better?